tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
```

Comment:
We want to use `esnext` here because we don't want the javascript files to be transpliled for older version, electron always supports newer javascript syntax.

Reply:
ok sure!